src/net.cpp

```diff
@@ -2449,12 +2449,6 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
         LOCK(cs_totalBytesRecv);
         nTotalBytesRecv = 0;
```

Comment:
Any reason to not move this as well?

Reply:
No reason to move it: it's already initialized to 0 at declaration, so it should already be safe; and moving it wouldn't reduce the number of locks. No reason not to move it either though, I think.

Reply:
Moved others to initialize at declaration, and no longer reinitializing any of them in Start()